src/ansys/geometry/core/tools/prepare_tools.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (C) 2023 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""Provides tools for preparing bodies."""
+
+from ansys.api.geometry.v0.bodies_pb2 import (
+    GetRequest,
+    GetType,
+)
+from ansys.api.geometry.v0.bodies_pb2_grpc import BodiesStub
+from ansys.api.geometry.v0.preparetools_pb2 import (
+    ShareTopologyRequest,
+)
+from ansys.api.geometry.v0.preparetools_pb2_grpc import PrepareToolsStub
+from beartype.typing import TYPE_CHECKING, List
+from google.protobuf.wrappers_pb2 import BoolValue, DoubleValue
+
+from ansys.geometry.core.connection import GrpcClient
+from ansys.geometry.core.typing import Real
+
+if TYPE_CHECKING:  # pragma: no cover
+    from ansys.geometry.core.designer.body import Body
+
+
+class PrepareTools:
+    """Prepare tools for PyAnsys Geometry."""
```

Comment:
```suggestion
    """Prepares tools for PyAnsys Geometry."""
```
Class descriptions start with a verb that ends in "s" or "es"

Reply:
Changed to "Exposes body preparation tools"